experiments/dsl/src/main/scala/org/apache/gearpump/streaming/dsl/plan/Planner.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.streaming.dsl.plan
+
+import akka.actor.ActorSystem
+import org.apache.gearpump.partitioner.{CoLocationPartitioner, HashPartitioner, Partitioner}
+import org.apache.gearpump.streaming.TaskDescription
+import org.apache.gearpump.streaming.dsl.op._
+import org.apache.gearpump.util.Graph
+
+import scala.collection.JavaConverters._
+
+class Planner {
+
+  /*
+   * Conert Dag[Op] to Dag[TaskDescription] so that we can run it easily.
+   */
+  def plan(dag: Graph[Op, OpEdge])(implicit system: ActorSystem): Graph[TaskDescription, Partitioner] = {
```

Comment:
Yes, it is covered by https://github.com/intel-hadoop/gearpump/pull/632/files#diff-fddec5f422f7b7e7c487cc6034cfa8d5R69